src/controller/error/error.controller.ts

```diff
@@ -6,5 +6,5 @@ export function notFoundErrorHandler(req: Request, res: Response): void {
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export function serverErrorHandler(err, req: Request, res: Response): void {
-  return res.status(err.statusCode || 500).render('error-pages/500');
+  return res.status(err?.statusCode || 500).render('error-pages/500');
```

Comment:
What does ?. do?

Reply:
This is just checking if the error is defined, if it's undefined, it will not check the status code, it will return undefined